components/theme.ts

```diff
@@ -55,6 +55,14 @@ export const theme = extendTheme({
     Heading: {
       baseStyle: {
         color: 'Gray.900',
+        fontWeight: 'semibold',
+      },
+    },
+    Button: {
+      baseStyle: {
+        bg: 'Primary.50',
+        background: 'Primary.50',
+        color: 'Primary.700',
       },
```

Comment:
For whatever reason, [it seems like we have to set both `bg` and `background` (in this order)](https://github.com/chakra-ui/chakra-ui/issues/3293#issuecomment-775081618) for it to work as expected.